frontend/src/app/components/modals/confirm-dialog/confirm-dialog.modal.ts

```diff
@@ -45,7 +45,8 @@ export interface ConfirmDialogOptions {
 }
 
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><del><span data-href="https://github.com/opf/openproject/blob/8bc34ee7fd4c0689c408c4b07a0bef8c4c9800ff/frontend/src/app/components/modals/confirm-dialog/confirm-dialog.modal.ts#L46"></span>Please use <code>ChangeDetectionStrategy.OnPush</code> for this component</del></td>
    </tr>
  </tbody>
</table>